### ISSUE TITLE
How do I create the victronenergy/venus-docker-server  container on a standalone setup (not using docker!)

### ISSUE THREAD

**Issue by virtuvas** [victronenergy/venus-docker-grafana#14]
evening all,

not sure I'm in the right place (although I've searched a lot!), so pointers welcomed.

I'll try to describe briefly what I want to achieve, but first where I stand:

- raspberrypi with CANHAT on board my motorboat.
- slow and expensive internet link sends data to VRM
- docker at home on win10 pc, picks VRM data presents them via grafana (you know, configured via http://localhost:8088/admin/#/settings/vrm to connect to VRM)

---all above done works fine---

Now, I set up an influx/signalK/grafana standalone NOT docker on my office PC (also running win10pro)
all's working fine as well. BUT for various reasons, as I said, it's NOT on docker, just standalone 64bit win apps, services running, etc.

due to bandwidth issues I'm trying to:
A. grab the data already sent from boat to VRM like I can at home with venus-docker-grafana, thus avoid extra data charges for sending more detailed data -which I don't really want!- to office pc.
B. send the NMEA2000 data I select, at varying rates/granularity to the influxDB on the office PC. (like depth, temp, humidity, etc every 15mins, generator, switch states, bilge pump every 30sec, tank levels every half hour, etc)

OK, B is a q for a later point once I figure out how I can configure different packets of info to be sent from the boat at different intervals.

I cannot seem to be able to do A which was straight forward in docker. I hope there is a way, hence my asking!
So, simple Q (I hope!), how can I create the Container: victronenergy/venus-docker-server at port 8088 on a standalone setup???
In other words, how can a standalone signalk setup connect and access VRM data?

cheers

Vassilis

**Comment by mman** [comment]
Not sure if this issue is still valid, but for the interested readers. `venus-server` (to be replaced with https://github.com/victronenergy/venus-influx-loader) can be configured to fetch data directly from Venus OS devices on the same network, or by talking to VRM. It does not need to be running in docker, once the server is setup, you simply access it on port 8088 to configure what Venus OS devices to fetch the data from (including the VRM) and where to store them. That is all. Feel free to comment to get more details.

**Comment by mman** [comment]
Please followup in the new repo https://github.com/victronenergy/venus-influx-loader if necessary. Closing here. @dirkjanfaber please close this issue.
